packages/manager/modules/telecom-spare/src/index.js

```diff
@@ -0,0 +1,75 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-mondial-relay';
+import '@ovh-ux/ng-ovh-contracts';
+import '@ovh-ux/ng-ovh-simple-country-list';
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'oclazyload';
+import 'ovh-ui-angular';
+import 'angular-leaflet-directive';
+import 'angular-ui-bootstrap';
+import 'angular-translate-loader-partial';
+import 'angular-translate-loader-pluggable';
+import 'angular-translate-storage-cookie';
+import 'angular-translate-storage-local';
+
+import 'ovh-manager-webfont/dist/css/ovh-font.css';
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css';
+import 'font-awesome/css/font-awesome.css';
+import 'ovh-common-style/dist/ovh-common-style.css';
+import 'leaflet/dist/leaflet.css';
+import 'ui-select/dist/select.css';
```

Comment:
Why is this needed ? Isn't it imported by UIKit ?

Reply:
I need them to display the map for order modem and phone